wallet-new/integration/Types.hs

```diff
@@ -32,6 +32,7 @@ type WalletTestMode m =
     ( MonadIO m
     , MonadThrow m
     , MonadPlus m
+    , MonadMask m
```

Comment:
Why `MonadMask` here?

Reply:
See my description:
> I am not happy because I had to add MonadMask constraint into WalletTestMode. I am not sure how to avoid that. The idea was to create a temporary file with withSystemTempFile dump poor key content into that file and do tests with import key.

I would like to remove that. Maybe you can help me remove `MonadMask` here. I didn't figure out how to do it.

Reply:
This should do it - https://github.com/input-output-hk/cardano-sl/pull/2670#discussion_r175061591